utils/s2n_init.c

```diff
@@ -22,8 +22,8 @@
 #include "utils/s2n_mem.h"
 #include "utils/s2n_random.h"
 #include "utils/s2n_safety.h"
-
```

Comment:
nit: Any reason to remove empty line? Seems like intention was to separate includes from different modules.

Reply:
I will add the line back

Reply:
still there?

Reply:
This is wired. I added line...